test/cljs/status_im/test/runner_macros.clj

```diff
@@ -0,0 +1,22 @@
+(ns status-im.test.runner-macros
+  (:require [cljs.analyzer.api :as ana-api]))
+
+(def cljs-tests-ordered-ns (->> ['status-im.test.utils.async]
```

Comment:
why is this one left explicitely ?

Reply:
Tests were failing with random execution order (I pasted the error into the PR), so I tried to reproduce the execution order that was working in the current runner. I don't know why the order is needed though, so if anyone has any clue, that could help detail why and maybe improve the sorting logic, or even get rid of the order requirement for tests to pass.